src/Analyzers/Async/src/Microsoft.Dotnet.Analyzers.Async.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Analyzer package for async best practices.</Description>
+    <VersionPrefix>$(ExperimentalVersionPrefix)</VersionPrefix>
+    <VersionSuffix>$(ExperimentalVersionSuffix)</VersionSuffix>
+    <PackageVersion>$(ExperimentalPackageVersion)</PackageVersion>
+    <InformationalVersion>$(ExperimentalPackageVersion)</InformationalVersion>
+    <VerifyVersion>false</VerifyVersion>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <EnableApiCheck>false</EnableApiCheck>
+    <BuildOutputTargetFolder>analyzers/dotnet/cs/</BuildOutputTargetFolder>
+    <UseLatestPackageReferences>true</UseLatestPackageReferences>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="AsyncAnalyzer.csv" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="AsyncAnalyzer.csv" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis.Common" PrivateAssets="All" />
+    <Reference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="All" />
+    <Reference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" PrivateAssets="All" />
+    <PackageReference Update="NETStandard.Library" PrivateAssets="All" />
```

Comment:
Why is this needed?

Reply:
I remember it being required at one point.

Reply:
Still required. I recommend keeping this.

The reason is that this assembly compiles for netstandard1.3, but it's not really a .NET Standard 1.3 project. Without adding PrivateAssets='all' to suppress this dependency, nuget generates an incorrect nuspec.

An alternative which is easier to grok is
```xml
    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
```